### PR TITLE
Storefront profile fix, Toast color 

### PR DESCRIPTION
**Pull request by sakibbye**
# Description

(Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.)

## Type of change

(Please delete options that are not relevant.)

- [x] Link to a **GH Issue** or **Jira** task here for reference.
- [x] Bug fix
- [ ] New feature

# How Has This Been Tested?

(Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration)

- [x] Tested by myself
- [ ] Unit test

# Developer Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# Review check list:

- [ ] Separation of Concerns followed (Reviewer name/NaN)
- [ ] Code is in sync with existing code patterns/technologies. (Reviewer name/NaN)
- [ ] Are there any obvious optimizations that will improve performance? (Reviewer name/NaN)
- [ ] Can any logging or debugging code be removed? (Reviewer name/NaN)
- [ ] Are all data inputs checked (for the correct type, length, format, and range) and encoded? (Reviewer name/NaN)
- [ ] Are invalid parameter values handled? (Reviewer name/NaN)
- [ ] Is error handling done the correct way? (Reviewer name/NaN)
- [ ] Are all the edge cases (null, 0, negative) handled properly? (Reviewer name/NaN)

### CODE REVIEW

**Comment by sakibbye** [comment]
https://pm23.atlassian.net/browse/BC-433
https://pm23.atlassian.net/browse/BC-431
https://pm23.atlassian.net/browse/BC-460

File: packages/storefront/components/myAccount/account-details.tsx [hunk 1]
```diff
@@ -15,6 +15,9 @@ import { CustomerSchema } from './schema/customer.schema';
 import Breadcrumb from '@/components/global/breadcrumbs/breadcrumb';
 import AccountDetailsForm from '@/components/myAccount/account-detailForm';
 import SingleDetail from '@/components/myAccount/singleDetail';
+import WithAuth from '@/components/auth/withAuth';
+
+var cookie = require('cookie');
```
Comment (asaduzzamanbs23) [inline packages/storefront/components/myAccount/account-details.tsx:20]: @sakibbye why you need cookie here

File: packages/storefront/components/myAccount/account-details.tsx [hunk 2]
```diff
@@ -49,6 +52,9 @@ const AccountDetails: React.FC = () => {
     dispatch(storeCustomerDetails(response!.data));
     setEditable(false);
   };
+  
+
+
```
Comment (asaduzzamanbs23) [inline packages/storefront/components/myAccount/account-details.tsx]: @sakibbye remove unusual line

File: packages/storefront/components/myAccount/account-details.tsx (hunk 1 above)
Comment (asaduzzamanbs23) [inline packages/storefront/components/myAccount/account-details.tsx]: @sakibbye why you need cookie here

**Review by asaduzzamanbs23 — CHANGES_REQUESTED** [review]
@sakibbye 
1. Remove unusual line
2. Remove unusual import

**Comment by sakibbye** [comment]
@asaduzzamanbs23 changes done bhai

**Review by asaduzzamanbs23 — APPROVED** [review]

**Pull request merged**